src/ng/sce.js

```diff
@@ -288,17 +322,23 @@ function $SceDelegateProvider() {
      * @name $sceDelegate#trustAs
      *
      * @description
-     * Returns an object that is trusted by AngularJS for use in specified strict
-     * contextual escaping contexts (such as ng-bind-html, ng-include, any src
-     * attribute interpolation, any dom event binding attribute interpolation
-     * such as for onclick,  etc.) that uses the provided value.
-     * See {@link ng.$sce $sce} for enabling strict contextual escaping.
+     * Marks the parameter as trusted for the specified context, and returns the
+     * trusted representation of it. This trusted object should later on be
+     * used as-is, without any security check, by sinks that use this context.
+     * For instance, marking a string as trusted for the $sce.HTML context will
+     * entirely bypass the potential $sanitize call in corresponding $sce.HTML
+     * sinks, such as an ng-bind-html directive, while using the same value
+     * passed as a string will cause it to be sanitized.
      *
-     * @param {string} type The kind of context in which this value is safe for use.  e.g. url,
-     *   resourceUrl, html, js and css.
-     * @param {*} value The value that that should be considered trusted/safe.
-     * @returns {*} A value that can be used to stand in for the provided `value` in places
-     * where AngularJS expects a $sce.trustAs() return value.
+     * See {@link ng.$sceDelegate#getTrusted getTrusted} for the
+     * function that will consume those trusted values, and {@link ng.$sce $sce}
+     * for general documentation about strict contextual escaping.
+     *
+     * @param {string} type The kind of context in which this value is safe
+     *   for use.  e.g. url, resourceUrl, html, js and css.
```

Comment:
Should I do that for the whole file, while I'm at it? Do you have a style guide for doc?